node/hyperbahn/rate_limiting.js

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
:+1: @kriskowal , please move this out of hyperbahn, it's a general tchannel concern